testcafe/tests/helpers/visualreview-utils.js

```diff
@@ -0,0 +1,46 @@
+import VisualReview from 'visualreview-client'
+import { t } from 'testcafe'
+import { getNavigatorOs, getNavigatorName, getResolution } from './utils'
+
+//Put this const in travis after POC
+const VISUALREVIEW_INSTANCE = 'visualreview.cozycloud.cc'
+
+export class VisualReviewTestcafe extends VisualReview {
+  constructor(options) {
+    super(options)
+    this.options.protocol = 'https'
+    this.options.hostname = VISUALREVIEW_INSTANCE
+  }
+
+  async takeScreenshotAndReview(imageName) {
+    await t.takeScreenshot(imageName)
+
+    this.options.properties.os = await getNavigatorOs()
+    this.options.properties.browser = await getNavigatorName()
```

Comment:
We could set the os and the browser in the constructor, since they won't change over time?

Reply:
After a bit of research, no I cannot set the `os` and `browser` in the constructor.

I cannot even create an `initStart` function to call right after creating the vr object, as I'm doing it in a `.before` (So I have one run for one fixture!)

The before/after fixture hooks are executed before the first test/after the last test on the server side so you cannot use interactions with the `ClientFunction` in them. (And `getNavigatorOs`and `getNavigatorName` are both using `ClientFunction`